Setup.bat

```diff
@@ -85,16 +88,17 @@ call :MarkStartOfBlock "Create folders"
     )
 call :MarkEndOfBlock "Create folders"
 
+
 call :MarkStartOfBlock "Retrieve dependencies"
-    spatial package retrieve tools           schema_compiler-x86_64-win32               %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\tools\schema_compiler-x86_64-win32.zip"
-    spatial package retrieve schema          standard_library                           %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\schema\standard_library.zip"
-    spatial package retrieve worker_sdk      c-dynamic-x86-msvc_md-win32                %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86-msvc_md-win32.zip"
-    spatial package retrieve worker_sdk      c-dynamic-x86_64-msvc_md-win32             %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip"
-    spatial package retrieve worker_sdk      c-dynamic-x86_64-gcc_libstdcpp-linux       %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-gcc_libstdcpp-linux.zip"
-    spatial package retrieve worker_sdk      c-static-fullylinked-arm-clang_libcpp-ios  %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\worker_sdk\c-static-fullylinked-arm-clang_libcpp-ios.zip"
-    spatial package retrieve worker_sdk      core-dynamic-x86_64-linux                  %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\worker_sdk\core-dynamic-x86_64-linux.zip"
-    spatial package retrieve worker_sdk      csharp                                     %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\worker_sdk\csharp.zip"
-    spatial package retrieve spot            spot-win64                                 %PINNED_SPOT_VERSION%       "%BINARIES_DIR%\Programs\spot.exe"
+    spatial package retrieve tools           schema_compiler-x86_64-win32               %PINNED_CORE_SDK_VERSION%   "%CORE_SDK_DIR%\tools\schema_compiler-x86_64-win32.zip" %DOMAIN_ENVIRONMENT_VAR%

```

Comment:
Nit: Could you format these with spaces so `%DOMAIN_ENVIRONMENT_VAR%` is vertically aligned?